WordPress/src/test/java/org/wordpress/android/ui/reader/discover/interests/ReaderInterestsViewModelTest.kt

```diff
@@ -0,0 +1,3 @@
+package org.wordpress.android.ui.reader.discover.interests
+
+class ReaderInterestsViewModelTest
```

Comment:
We might want to consider initializing even the tests in this PR. Something like the following.  
```
@RunWith(MockitoJUnitRunner::class)
class ReaderInterestsViewModelTest {
@Rule
    @JvmField val rule = InstantTaskExecutorRule()

private lateinit var viewModel: ReaderInterestsViewModel

 @Before
    fun setUp() {
      viewModel = ReaderInterestsViewModel()
    }
}
```